### PR TITLE
Rename PodDisruptionsAllowed to DisruptionsAllowed in type PodDisruptionBudgetStatus

### PR DESCRIPTION
**Pull request by nan-yu**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

Update the field name to follow the same naming convention

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by nan-yu** [comment]
cc @mortent

**Review by draveness — COMMENTED** [review]
/priority backlog
/lgtm

This PR looks good to me since it doesn't break the API

**Comment by mortent** [comment]
This change was originally part of the PR to make PDBs GA. This moves it into a separate PR to reduce the size and complexity of the PDBs to GA PR ref https://github.com/kubernetes/kubernetes/pull/81571#discussion_r344813458. I will update #81571 when this is merged.

/lgtm
cc @lavalamp

File: staging/src/k8s.io/api/policy/v1beta1/types.go [hunk 1]
```diff
@@ -47,7 +47,7 @@ type PodDisruptionBudgetSpec struct {
 // PodDisruptionBudgetStatus represents information about the status of a
 // PodDisruptionBudget. Status may trail the actual state of a system.
 type PodDisruptionBudgetStatus struct {
-	// Most recent generation observed when updating this PDB status. PodDisruptionsAllowed and other
+	// Most recent generation observed when updating this PDB status. DisruptionsAllowed and other
```
Comment (lavalamp) [inline staging/src/k8s.io/api/policy/v1beta1/types.go]: We should actually use the json name in the comment (`disruptionsAllowed`); I like to use backticks to make it even more obvious and less weird to start the sentence with a lowercase letter.

**Comment by lavalamp** [comment]
LGTM - let me know what you want to do about the comment.

File: staging/src/k8s.io/api/policy/v1beta1/types.go (hunk 1 above)
Comment (nan-yu) [reply staging/src/k8s.io/api/policy/v1beta1/types.go]: Thanks for the comment. I checked comments for other fields in this file. We kind of use both ways, lowercase first letter (`// allowedUnsafeSysctls`) and uppercase first letter (`// AllowedCSIDrivers is`), but no backticks. 

Given this sentence is not the start of the comment, I am inclined to keep it as-is to make it more in-line with other comments

Comment (lavalamp) [reply staging/src/k8s.io/api/policy/v1beta1/types.go]: @liggitt we should probably document this more clearly-- looks like it's not covered by https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#naming-conventions, is it covered somewhere else?

Comment (lavalamp) [reply staging/src/k8s.io/api/policy/v1beta1/types.go]: Given that it's not officially documented I won't block the PR, I guess

(Note that it being wrong in other places isn't a reason I accept as valid for keeping it wrong here :) )

Comment (liggitt) [reply staging/src/k8s.io/api/policy/v1beta1/types.go]: hmm, not that I know of. need a godoc section in that doc, and an issue tracking sweeping existing types to fix docs

**Pull request merged**